solidity/random-beacon/hardhat.config.ts

```diff
@@ -112,28 +112,32 @@ const config: HardhatUserConfig = {
     },
     governance: {
       default: 1,
+      goerli: 0,
       // mainnet: ""
     },
   },
   external: {
-    contracts: [
-      {
-        artifacts:
-          "node_modules/@threshold-network/solidity-contracts/export/artifacts",
-        deploy:
-          "node_modules/@threshold-network/solidity-contracts/export/deploy",
-      },
-    ],
-    // deployments: {
-    //   // For hardhat environment we can fork the mainnet, so we need to point it
-    //   // to the contract artifacts.
-    //   hardhat: process.env.FORKING_URL ? ["./external/mainnet"] : [],
-    //   // For development environment we expect the local dependencies to be linked
-    //   // with `yarn link` command.
-    //   development: ["node_modules/@keep-network/keep-core/artifacts"],
-    //   ropsten: ["node_modules/@keep-network/keep-core/artifacts"],
-    //   mainnet: ["./external/mainnet"],
-    // },
+    contracts:
+      process.env.USE_EXTERNAL_DEPLOY === "true"
+        ? [
+            {
+              artifacts:
+                "node_modules/@threshold-network/solidity-contracts/export/artifacts",
+              deploy:
+                "node_modules/@threshold-network/solidity-contracts/export/deploy",
+            },
+          ]
+        : undefined,
+    deployments: {
+      // For hardhat environment we can fork the mainnet, so we need to point it
+      // to the contract artifacts.
+      hardhat: process.env.FORKING_URL ? ["./external/mainnet"] : [],
```

Comment:
Shouldn't we have some contracts (`T`?) in `solidity/random-beacon/external/mainnet/` for this to work?

Reply:
Ah, right... Then I think we should change the command we execute in the CI:
https://github.com/keep-network/keep-core/blob/77d7425e57e543581d069a5060f4ebcd3eec6dc0/.github/workflows/contracts-random-beacon.yml#L160

Reply:
38ce9988d2f0e710452fb9845aebc7c5af6a8018